x-pack/plugins/security_solution/public/timelines/components/timeline/body/index.tsx

```diff
@@ -225,60 +227,65 @@ export const BodyComponent = React.memo<StatefulBodyProps>(
       },
       [columnHeaders.length, containerRef, data.length]
     );
+    const kibana = useKibana();
+    const casesPermissions = useGetUserCasesPermissions();
+    const CasesContext = kibana.services.cases.getCasesContext();
 
     return (
       <>
         <TimelineBody data-test-subj="timeline-body" ref={containerRef}>
-          <EventsTable
-            $activePage={activePage}
-            $columnCount={columnCount}
-            data-test-subj={`${tabType}-events-table`}
-            columnWidths={totalWidth}
-            onKeyDown={onKeyDown}
-            $rowCount={data.length}
-            $totalPages={totalPages}
-          >
-            <ColumnHeaders
-              actionsColumnWidth={actionsColumnWidth}
-              browserFields={browserFields}
-              columnHeaders={columnHeaders}
-              isEventViewer={isEventViewer}
-              isSelectAllChecked={isSelectAllChecked}
-              onSelectAll={onSelectAll}
-              show={show}
-              showEventsSelect={false}
-              showSelectAllCheckbox={showCheckboxes}
-              sort={sort}
-              tabType={tabType}
-              timelineId={id}
-              leadingControlColumns={leadingControlColumns}
-              trailingControlColumns={trailingControlColumns}
-            />
+          <CasesContext owner={[APP_ID]} userCanCrud={casesPermissions?.crud ?? false}>
```

Comment:
This is the important change. To show the create case flyout, and eventually the case modal, the cases context needs initialization. Effectively this is the place where the flyout renders when invoked.

Reply:
Note: this is the table you see when you have alerts in a timeline in the timelines view.